src/components/15Years/15YearsButton.vue

```diff
@@ -0,0 +1,105 @@
+<script>
+export default {
+	render(createElement) {
+		const options = {
+			class: ['fifteen-yr-button', this.variant ? `fifteen-yr-button--${this.variant}` : ''],
+			props: {},
+			attrs: {},
+			on: { click: this.onClick },
+		};
+		if (this.to) {
+			options.props.to = this.to;
+		}
+		if (this.href) {
+			options.attrs.href = this.href;
+		}
+		return createElement(this.tag, options, this.$slots.default);
+	},
+	props: {
+		to: {
+			type: String,
+			default: null
+		},
+		href: {
+			type: String,
+			default: null
+		},
+		/**
+		 * Styling variant
+		 * `'', white, black`
+		* */
+		variant: {
+			type: String,
+			default: '' // default appears as mint
+		}
+	},
+	computed: {
+		tag() {
+			if (this.to) {
+				return 'router-link';
+			}
+			return this.href ? 'a' : 'button';
+		}
+	},
+	methods: {
+		onClick(e) {
```

Comment:
Testing this out. If it seems solid we can add to KvButton so in templates we can just write `<kv-button @click="myMethod">` and avoid the pitfall of forgetting `<kv-button @click.native="myMethod">`

Reply:
VUE-354

Reply:
👍  great improvement!